A job failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/8859514583/job/24330863340) [(plain)](https://github.com/rust-lang-ci/rust/commit/2c94366893e6e339fe024fc057b37c2f442a9fe3/checks/24330863340/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
env:
  TOOLSTATE_REPO: https://github.com/rust-lang-nursery/rust-toolstate
  TOOLSTATE_REPO_ACCESS_TOKEN: ***
##[endgroup]
/home/runner/work/_temp/06fd6a20-5201-4f3f-89ef-e7caa0eaf28b.sh: line 1: src/ci/publish_toolstate.sh: No such file or directory
##[error]Process completed with exit code 127.
```

</details>

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -177,6 +177,10 @@ jobs:
       - job
     if: "success() && github.event_name == 'push' && github.ref == 'refs/heads/auto' && github.repository == 'rust-lang-ci/rust'"
     steps:
+      - name: checkout the source code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
       - name: publish toolstate
         run: src/ci/publish_toolstate.sh
         shell: bash

--- a/src/ci/github-actions/ci.yml
+++ b/src/ci/github-actions/ci.yml
@@ -286,6 +286,10 @@ jobs:
     if: "success() && github.event_name == 'push' && github.ref == 'refs/heads/auto' && github.repository == 'rust-lang-ci/rust'"
     <<: *base-outcome-job
     steps:
+      - name: checkout the source code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
       - name: publish toolstate
         run: src/ci/publish_toolstate.sh
         shell: bash

